Dockerfile.mysql

```diff
@@ -0,0 +1,10 @@
+FROM mysql:latest
+
+ENV MYSQL_DATABASE=app_db
+ENV MYSQL_USER=app_user
+ENV MYSQL_PASSWORD=1234
+ENV MYSQL_ROOT_PASSWORD=root_password
+
+EXPOSE 3306
+
+VOLUME /var/lib/mysql
```

Comment:
Recommend to review Dockerfile structure (link to documentation: https://docs.docker.com/get-started/docker-concepts/building-images/writing-a-dockerfile/). You miss important part of Dockerfile
Please check Dockerfile examples for MySQL.